statsmodels/examples/try_gee.py

```diff
@@ -71,9 +71,10 @@
 print(bse)
 
 print("\nimplemented `standard_errors`")
-bse2 = np.column_stack((mdf2.bse, mdf2.standard_errors(),
-                                 mdf2.standard_errors(covariance_type='naive'),
-                                 mdf2.standard_errors(covariance_type='bias_reduced')))
+bse2 = np.column_stack((mdf2.bse,
+                        mdf2.standard_errors(),
+                        mdf2.standard_errors(cov_type='naive'),
+                        mdf2.standard_errors(cov_type='bias_reduced')))
```

Comment:
bias_reduced isn't a valid option.  Please advise.

Reply:
according to docs and code it is
https://www.statsmodels.org/dev/generated/statsmodels.genmod.generalized_estimating_equations.GEEResults.standard_errors.html

Reply:
```
$ python statsmodels/examples/try_gee.py
[...]
Traceback (most recent call last):
  File "statsmodels/examples/try_gee.py", line 77, in <module>
    mdf2.standard_errors(cov_type='bias_reduced')))
  File "statsmodels/genmod/generalized_estimating_equations.py", line 1826, in standard_errors
    "GEE: `bias_reduced` covariance not available")
ValueError: GEE: `bias_reduced` covariance not available
```

Reply:
that's different
AFAIU without looking: The traceback means that the bias_reduced covariance has not been computed, and doesn't mean that the option is not available.

I don't have time right now to check what triggers now the covariance computation.

Reply:
based on my reading of the code, there is no direct, public option/method anymore to get the additional covmat.
We changed this to lazy computation similar to other cov_type in fit cases

Reply:
Thanks for taking a look.  Does that mean that the example is correct and the gee code needs tweaking, the other way around, or both need tweaking?

BTW since you mentioned making doc notebooks as an Approved Activity the other day I'm hoping spending time on this counts.  Turns out troubleshooting these is still less frustrating than Bay Area apartment hunting, as long as its not actively antagonizing.

Reply:
I'm not sure yet whether or how we change the GEEResults for this.
`fit(cov_type=...)` is now our standard across models, but `standard_errors` and a "non-working" option is not.